test/verify/check-networking-firewall

```diff
@@ -177,6 +178,66 @@ class TestFirewall(MachineCase):
             b.click("caption button.btn-primary")
             b.wait_present("#cockpit_modal_dialog li input[data-id='empty']")
 
+    @skipImage("Custom ports were introduced in 191", "rhel-7-6-distropkg")
```

Comment:
Please refer to the PR #, not a version number -- it's not entirely clear when this will land, and PRs make backporting easier.